pkg/controller/pxc/controller.go

```diff
@@ -238,6 +238,8 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		return err
 	}
 
+	nodeSet.Spec.UpdateStrategy.Type = cr.Spec.UpdateStrategy
```

Comment:
for proxysql also

Reply:
Make this assignment as part of the function https://github.com/percona/percona-xtradb-cluster-operator/blob/master/pkg/pxc/statefulset.go#L14. That package made not to duplicate code between pxc and proxysql